### DISCUSSION TITLE
Unable to Remove Photos from EOS v7.1

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#577] [Q&A] [answered]
Just did a reinstall of v7.1 on my ancient Dell XPS laptop.

I've tried deleting photos from the AppCenter and it won't go away.

I tried sudo apt-get remove io.elementary.photos, but it comes back as not found.

Doing a apt list | grep -i 'photos' results in this:

```
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

gnome-photos-tests/jammy 42.0-1 amd64
gnome-photos/jammy 42.0-1 amd64
pantheon-photos-common/jammy 2.8.0+r4580+pkg64~ubuntu7.1 amd64
pantheon-photos/jammy,now 2.8.0+r4580+pkg64~ubuntu7.1 amd64 [installed,automatic]
ubuntu-mate-wallpapers-photos/jammy 22.04.17 all
unity-lens-photos/jammy 1.0+17.10.20170605-0ubuntu9 all


```
Which one is the one I want to remove?

Help!

**Comment by ryonakano** [comment, marked as answer]
As you can see the result of apt list, the package name of Photos is `pantheon-photos`, not `io.elementary.photos`.
